lib/ui/semantics.dart

```diff
@@ -307,6 +309,28 @@ class SemanticsFlag {
   /// is a password or contains other sensitive information.
   static const SemanticsFlag isObscured = const SemanticsFlag._(_kIsObscuredIndex);
 
+  /// Whether the semantics node is the root of a subtree for which values
+  /// should be announced.
+  /// 
+  /// When a semantics node with a node flag is added to the tree, the
+  /// framework will search for the first child in inverse hit-test order with
+  /// a routeName flag and non-empty semantic value and announce it as an edge
+  /// transition.
+  /// 
+  /// This is used in widgets such as Routes, Drawers, and Dialogs to 
+  /// communicate significant changes in the visible screen.
+  static const SemanticsFlag isRoute = const SemanticsFlag._(_kIsRouteIndex);
+
+  /// Whether the semantics node value is the name of a visually distinct route.
+  /// 
+  /// This is used by certain widgets like Drawers and Dialogs, to indicate
+  /// that the node's semantic value can be used for an edge triggered
```

Comment:
And is it value or value label hint?

Also, should it be label instead of value? I think the app bar title is transfered as a label, right?

Reply:
The app bar title is a label and not a value - I decided to change it since I noticed that android accessibility bridge sometimes combined the three?

I'm not 100% on the semantic differences between label and value.  If label is a better fit then I can do a quick update

Reply:
I believe label is the more correct one.

Label is meant for static things (like the title of an app bar). Value is for dynamic, often changeable things like the content of a text fields.

Android doesn't have a way to separate between the two, that's why we often munch them together. However, in this case it sounds like only label is the right choice for both platforms.

Reply:
Got it, that makes more sense given that we don't make additional announcements if the route name changes too